tracing/tracing.go

```diff
@@ -82,13 +84,38 @@ func (o SpanOptions) spanLabel() string {
 	return fmt.Sprintf("%s::%s", o.Namespace, o.label)
 }
 
-// StartSpan creates a new span
+// StartSpan creates a new span (DEPRECATED : use Span instead)
 func StartSpan(ctx context.Context, label string, funcs ...func(*SpanOptions)) (context.Context, func()) {
+	fs := []func(*SpanOptions){Label(label)}
```

Comment:
Can we do something annoying on this call?

Reply:
what do yoy mean ?

Reply:
I was wondering if there is a way to incentivize developers to migrate. Only spawning a warning comes to mind.

Reply:
The only annoying thing would be to completely remove the function and bump the version to make the build fail. That's kinda annoying :D 

TBH I think I'll do that after a few month. The migration is fairly easy (I've done it as a test on credits it took me 2min for the full service; a bigger service would take maybe one hour to migrate).

Reply:
Then adding a warning with upper counter wouldn't hurt.